dvc/project.py

```diff
@@ -6,6 +6,8 @@
 import os
 import dvc.prompt as prompt
 import dvc.logger as logger
+from dvc.stage import StageFileDoesNotExistError
```

Comment:
Let's import this right in the `checkout()`. `stage` module has some heavy dependencies that slow down dvc calls, so we try to do dynamic imports where we can.